OSDOCS-4920: adding content about limit message (#58414)
openshift/openshift-docs: web_console/customizing-the-web-console.adoc

include::modules/customizing-the-web-console-URL.adoc[leveloffset=+1]

> == Recognizing resource and project limits and quotas
> 
> A message appears in the *Topology* or *List* views accessible through the web console, when a deployment reaches its preset limits or quotas. 
> 

Reviewer: ```suggestion
[NOTE]
====
When a deployment reaches its preset limits or quotas, a message appears in the *Topology* and *List* views of the web console.
====

```

Author: Left as a separate section because I increased the amount of content to resolve other comments.

include::modules/customizing-the-web-console-URL.adoc[leveloffset=+1]

[NOTE]
====

> When a deployment reaches its preset limits or quotas, a message appears in the *Topology* and *List* views of the web console.

Reviewer: Per Andrea's comment, I'd also address _where_ the "Topology" and "List" views are, e.g:

"When a deployment reaches its preset limits or quotas, a message appears in the **Topology** view of the **Developer** perspective, or in the **List** view of X in the web console."

I don't know where the List view is or what this is referring to.

Author: did not make an admonition because of text expansion.

include::modules/customizing-the-web-console-URL.adoc[leveloffset=+1]

[id="customizing-web-console-recognize"]
== Recognizing resource and project limits and quotas

> When in Developer mode of the web console, when in the *Topology* view, a graphical representation of available resources appears. A yellow border appears around the resource name indicating a message regarding resource limitation or quotas being reached. Click the resource to open a side panel to see the message. If the *Topology* view has been zoomed out, a yellow dot indicates that a message is provided.

Reviewer: Please use "**Developer** perspective" instead of "Developer mode" here to keep consistency with other docs about dev console. See https://docs.openshift.com/container-platform/4.13/applications/creating_applications/odc-creating-applications-using-developer-perspective.html for examples

Author: done

> When in Developer mode of the web console, when in the *Topology* view, a graphical representation of available resources appears. A yellow border appears around the resource name indicating a message regarding resource limitation or quotas being reached. Click the resource to open a side panel to see the message. If the *Topology* view has been zoomed out, a yellow dot indicates that a message is provided.
> 
> If using *List View* from the *View Shortcuts* menu. Resources appear as a list. An *Alerts* column indicates if a message is available. 
> 

Reviewer: ```suggestion
You can view a graphical representation of available resources in the *Topology* view of the web console *Developer* perspective.

If a resource has a message about resource limitations or quotas being reached, a yellow border appears around the resource name. Click the resource to open a side panel to see the message. If the *Topology* view has been zoomed out, a yellow dot indicates that a message is available.

If using *List View* from the *View Shortcuts* menu, resources appear as a list. The *Alerts* column indicates if a message is available. 

```

some suggested rewording for clarity

Author: done
